app/actions/__tests__/unitActions.spec.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from 'chai';
+import simple from 'simple-mock';
+
+import * as apiUtils from 'utils/apiUtils';
+import { fetchUnits } from 'actions/unitActions';
+import types from 'constants/ActionTypes';
+
+describe('Actions: unitActions', () => {
+  let getRequestTypeDescriptorMock;
+  let buildAPIUrlMock;
+
+  beforeEach(() => {
+    getRequestTypeDescriptorMock = simple.mock(apiUtils, 'getRequestTypeDescriptor');
+    buildAPIUrlMock = simple.mock(apiUtils, 'buildAPIUrl');
+  });
+
```

Comment:
I suggest restoring mocked functions after each unit test

```suggestion

    afterEach(() => {
      simple.restore();
    });
```

Reply:
Ah, i forgot that! 👍